storm-core/src/clj/backtype/storm/ui/core.clj

```diff
@@ -510,7 +501,7 @@
                              (map #(.get_num_executors ^TopologySummary %))
                              (reduce +))]
        {"user" user
-        "stormVersion" (read-storm-version)
+        "stormVersion" (str (VersionInfo/getVersion))
```

Comment:
Now that we have more information then just the version number, it would be great to expose that too.  If you don't want to do it just yet I understand, we can file a follow up JIRA to do the work.